core/ui/TiddlerTemplate.tid

```diff
@@ -1,3 +1,3 @@
 title: $:/core/ui/TiddlerTemplate
 
-<$transclude tiddler={{{ [<currentTiddler>] :cascade[all[shadows+tiddlers]tag[$:/tags/TiddlerTemplateFilter]get[text]] :and[!is[blank]else[$:/core/ui/ViewTemplate]] }}} />
+<$transclude tiddler={{{ [<currentTiddler>] :cascade[all[shadows+tiddlers]tag[$:/tags/TiddlerTemplateFilter]get[text]] :and[!is[blank]else{$:/core/ui/ViewTemplate}] }}} />
```

Comment:
typo:
{$:/config/ui/ViewTemplate}, not $:/core

Reply:
Thanks @saqimtiaz, fix upcoming